python/flask/webappWithTests/Tests/functional_tests/test_functional.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from selenium import webdriver
+import unittest
+import os
+import sys
+import pytest
+
+class FunctionalTests(unittest.TestCase):
+
+	def setUp(self):
+		options = webdriver.ChromeOptions()
+		options.add_argument('--no-sandbox')
+		self.driver = webdriver.Chrome(os.environ["ChromeWebDriver"], chrome_options=options)
+
+	"""
+    Sample selenium test. Asserting webapp page title and since webapp deployment takes
```

Comment:
Can you change it to:

The current time taken by the webapp to refresh after deployment is a considerable amount and the selenium tests in the release run much faster than this duration, hence the tests do not assert the current deployed app but from the last deployment (in case of the first deployment it happens to be the default iis page). Hence the try catch around the title assertion which is a temporary solution until the webapp deployment refresh times are fixed.

Reply:
Changes the comment